docs/modules/user-guide/partials/proc_adding-container-component-to-a-devfile.adoc

```diff
@@ -25,12 +25,12 @@ A component type that allows to define a container image-based configuration of
         - name: mavenrepo
           path: /root/.m2
        env:
-         - name: ENV_VAR
+         - name: env_var
            value: value
        endpoints:
          - name: maven-server
            targetPort: 3101
-           configuration:
+           exposure:
```

Comment:
`exposure` level should be removed. `protocol` and `secure` have moved up one level (same level as targetPort). scheme has been removed.

So the example will looks like:
```
           protocol: https
           secure: 'true'
```